src/js/visual/configurator-csr.js

```diff
@@ -1741,6 +1741,7 @@ ripe.ConfiguratorCsr.prototype._onPostConfig = async function(self, config) {
 
         self.loading = false;
         this.trigger("ready");
+        this.trigger("post_config", config);

```

Comment:
Won't this do a infinite loop?
This method is run as a callback for `post_config` event and not doing the config itself.

Reply:
I get what you're saying because of 
```js
ripe.ConfiguratorCsr.prototype._registerConfigHandlers = function() {
    this.owner.bind("pre_config", (brand, model, options) => this._onPreConfig(this));
    this.owner.bind("post_config", config => this._onPostConfig(this, config)); // <-- this
};
```

It doesn't do an infinite loop because `this.trigger` triggers a configurator event which is different from `this.owner.trigger`.

I do recognize the name of the event is kind of generic but I wasn't sure of the best option since I also wanted to keep it generic and not something like `post_config_csr`.

Do you have any suggestion?

Reply:
Hmm 🤔 
So the idea is to know when the configurator is done loading right? If that's the case I would use the `ready` event and just make it have params and one of the params being a ID of the stuff we are tracking

Reply:
Added `{ id: "configurator_csr" }` inspired by the name of the context.
Now I'm not sure this is the best structure. Maybe I'd prefer `{ origin: "configurator_csr" }` instead. What do you say?

Reply:
I guess it depends on what is the purpose of the variable. If it's to inform just origin of the event, maybe it makes sense. Even a specified event for that. If it's to differentiate configurators, I guess an ID is better